Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIScriptingCocoa.mm

```diff
@@ -117,7 +117,7 @@
         auto injectedFrames = parameters.frameIDs ? WebCore::UserContentInjectedFrames::InjectInTopFrameOnly : WebCore::UserContentInjectedFrames::InjectInAllFrames;
 
         auto styleSheetPairs = getSourcePairsForParameters(parameters, *this);
-        injectStyleSheets(styleSheetPairs, webView, *m_contentScriptWorld, parameters.styleLevel, injectedFrames, *this);
+        injectStyleSheets(styleSheetPairs, webView, *RefPtr { m_contentScriptWorld }, parameters.styleLevel, injectedFrames, *this);
```

Comment:
`m_contentScriptWorld` is already a `RefPtr`.

Reply:
That's true--but per the C++ safe guidelines, we want to keep a smart pointer of an object on the stack when passing the object to a non-trivial function.

Reply:
Per safe C++ guidelines, we indeed need a smart pointer *on the stack*. The data member could get reassigned at any point and cause use after frees.

However, I think I would have written this as `Ref { *m_contentScriptWorld }`

Reply:
Fixed!